cumulusci.yml

```diff
@@ -341,6 +341,164 @@ tasks:
                 - "connection was cancelled here"
             retry_always: True
 
+    cleanup_translation_metadata:
+        description: Delete extraneous and translation-excluded metadata from translation files after using retrieve_unpackaged to pull down cleaned translations from a scratch org.
+        class_path: tasks.modify.RemoveElementsXPath
```

Comment:
@force2b  Does this need to be changed back to the core CCI class?

Reply:
Good question. I thought that was the core class.

Reply:
The core CCI class is:

cumulusci.tasks.metadata.modify.RemoveElementsXPath

I "forked" it for you so you wouldn't need to wait until CCI included a version with the delete feature.

Reply:
Ah. Thanks. I'll fix here and in all the underlying repo's as well.